docs/vo_service_discovery/vo_service_discovery.rst

```diff
@@ -0,0 +1,202 @@
+.. doctest-skip-all
+
+.. _astroquery.vo_service_discovery:
+
+*************************************************************************
+Virtual Observatory Service Discovery (`astroquery.vo_service_discovery`)
+*************************************************************************
+
+Getting Started
+===============
+
+Archives from around the world have registered their services with the NASA
+Astronomy Virtual Observatory (NAVO) directory. A web interface is available
+at http://vao.stsci.edu/keyword-search/. This module provides methods
+to query this directory by service type, keywords, wavelength band or coverage,
+and source or publisher.
+
+Here is an example showing how to find information about conesearch services
+provided by the NASA Extragalactic Database (NED).
+
+.. code-block:: python
+
+    >>> from astroquery.vo_service_discovery import Registry
```

Comment:
One big-picture suggestion: can we either rename the module `VORegistry` or suggest a user standard of doing that?   e.g., `from astroquery.vo_service_discovery import Registry as VORegistry`?  I don't like the idea of user code existing in the wild with the generic name `Registry`, since that could be a registry for anything.